tests/test_lab/test_gates_fock.py

```diff
@@ -161,14 +161,14 @@ def test_parallel_displacement(x1, x2, y1, y2):
 
 def test_squeezer_grad():
     """tests fock squeezer gradient against finite differences"""
```

Comment:
We can remove this line with the more explicitely-named test

Reply:
if you mean the docstring we can't remove it because codefactor wants a docstring and it won't let you merge

Reply:
Afaik tests don't canonically need a docstring. When names are explicit enough they don't need one (afaik, again!).
Codefactor is a tool, it can be parameterised! If we have to do stuff that doesn't make sense to please a tool, the tool is wrong :,)